src/RFC-0131_Mining.md

```diff
@@ -74,52 +74,33 @@ independent, then one can get at most x% of the total hash rate, where x is the
 on CPUs, GPUs, and ASICs. In truth, the distinction between GPUs and ASICs is only a matter of time. Any "GPU-friendly"
-algorithm is ASIC-friendly too; it's just a case of whether the capital outlay for fabricating them is worth it; and
-this will eventually become true for any algorithm that supplies PoW for a growing market cap. Employing merged mining
-with major players that use independent hardware introduces another degree of freedom, as long as those are independent,
-like RandomX with Monero, SHA-256 with Bitcoin and Scrypt with Litecoin.
-
-_**Note:** Merge mining does not add security per se, but it does add plenty of hash rate and continuity of the blockchain._
-
-So really the answer to how many algorithms is: More than one, as independent as possible.
-
-### The choice of algorithms
+algorithm is ASIC-friendly, too; it's just a case of whether the capital outlay for fabricating them is worth it, and
+this will eventually become true for any algorithm that supplies PoW for a growing market cap.
 
-A good technical choice would be merge mining with Monero, Bitcoin and Litecoin, if enough interest could be
-attracted from those mining communities. However, that would rule out any participation from Tari supporters and
-enthusiasts, at least in the early stages. So, to be inclusive of Tari supporters and enthusiasts, merge mining RandomX
-with Monero and another GPU/ASIC-friendly algorithm, like SHA3 also known as Keccak, is proposed. Using a custom
-configuration of such a simple and well understood algorithm means there is a low likelihood of unforeseen optimizations
-that will give a single miner a huge advantage. It also means that it stands a good chance of being "commoditized" when 
-ASICs are eventually manufactured. This means that SHA3 ASICs will be widely available and not available from only a 
-single supplier.
+With this in mind, we should only choose one GPU/ASIC algorithm and one for CPUs. 
 
-_**Edit:** Handshake, which launched a few months ago, selected a Hashcash PoW algorithm
-[(see #Consensus)](https://handshake.org/files/handshake.txt) using SHA3 and Blake2B for many of the same reasons:
-SHA3 is currently under-represented in PoW; SHA3 usage in combination with Blake2B in PoW creates a more level playing
-field for hardware manufacturers._
+An excellent technical choice would be merge mining with Monero using RandomX as a CPU-only algorithm and SHA3, also known as Keccak
+, for a GPU/ASIC-friendly algorithm. Using a custom configuration of such a simple and well-understood algorithm means there is 
+a low likelihood of unforeseen optimizations that will give a single miner a considerable advantage. It also means that it stands a 
+good chance of being "commoditized" when ASICs are eventually manufactured. This means that SHA3 ASICs will be widely available 
+and not available from only a single supplier.
```

Comment:
```suggestion
good chance of being "commoditized" when ASICs are eventually manufactured. This would mean that SHA3 ASICs are widely available from multiple suppliers.
```